moped-editor/src/layouts/DashboardLayout/NavBar/NavigationSearchResults.js

```diff
@@ -55,13 +55,13 @@ const NavigationSearchResults = ({
         </ListItemLink>
       ))}
       {results.length > 5 && (
+        // Link to project list view with simple search populated
         <>
           <Divider />
           <ListItemLink
-            to={`/moped/projects/`}
-            // send searchTerm in location state
-            state={{ searchTerm: searchTerm }}
+            to={`/moped/projects?search=${searchTerm}`}
             onClick={handleDropdownClose}
+            reloadDocument
```

Comment:
This forces a browser redirect because client-side navigation doesn't re-render the project list view. That render is needed to initialize the search term from the url param. I think this is alright, but I'm open to other suggestions!

Reply:
cool—i'm glad you thought of this feature! this works as i expect it to, and i think forcing a re-render makes sense.

i'm not sure what an alternative might be. maybe a a global search param context that would cause the consumers to re-render?

anyway, this looks good to me!